unittests/runtime-ffi/ffi.cpp

```diff
@@ -453,40 +454,44 @@ BOOST_AUTO_TEST_CASE(call_variadic) {
 }
 
 BOOST_AUTO_TEST_CASE(alloc) {
-  mpz_t s1;
+  mpz_t s1, s2;
```

Comment:
Would it be hard to also add a test for alignment?

Reply:
I mean, I'm not sure that we could really test something reliably. We could test that the address returned is indeed aligned by a certain amount, but the actual address returned is different every time we run the unit tests, so it seems sketchy to me to make the tests depend on the address like that. Besides, the implementation is allowed to return a value that is aligned by a greater amount than specified by the user, which means that even if the test passed, that doesn't really prove there's not a bug.

Reply:
Hm, ok, maybe it's fine to not have a test, but I think that if you allocate 1 byte with a 32k alignment repeatedly without freeing the previous memory (say, 10 times, or 100 times if you really want to make sure), then I'd say that's (really) unlikely to pass unless the hook actually aligns things.